src/net/sf/memoranda/ui/EventDialog.java

```diff
@@ -546,7 +547,14 @@ public void noRepeatRB_actionPerformed(ActionEvent e) {
     }
 
     void okB_actionPerformed(ActionEvent e) {
-        this.dispose();
+    	if (textField.getText().isEmpty())
+    	{
+    		JOptionPane.showMessageDialog(eventPanel, "Detail cannot be empty.");
```

Comment:
I'll check - I thought for loops it was specified that braces would be on a line.



On Wed, Feb 10, 2016 at 12:01 AM -0800, "krcarey1" <notifications@github.com<mailto:notifications@github.com>> wrote:


In src/net/sf/memoranda/ui/EventDialog.java<https://github.com/cst316/spring16project-Team-El-Paso/pull/19#discussion_r52424596>:

> @@ -546,7 +547,14 @@ public void noRepeatRB_actionPerformed(ActionEvent e) {
>      }
>
>      void okB_actionPerformed(ActionEvent e) {
> -        this.dispose();
> +     if (textField.getText().isEmpty())
> +     {
> +             JOptionPane.showMessageDialog(eventPanel, "Detail cannot be empty.");


I thought we were doing { at the end of a line and } on its own line per the CG?

-
Reply to this email directly or view it on GitHub<https://github.com/cst316/spring16project-Team-El-Paso/pull/19/files#r52424596>.